src/cpp/session/modules/SessionFind.cpp

```diff
@@ -909,11 +909,15 @@ class GrepOperation : public boost::enable_shared_from_this<GrepOperation>
    bool shouldSkipFile(std::string file)
    {
       return (file.find("/.Rproj.user/") != std::string::npos ||
+              file.find("/.quarto/") != std::string::npos ||
               file.find("/.git/") != std::string::npos ||
               file.find("/.svn/") != std::string::npos ||
               file.find("/packrat/lib/") != std::string::npos ||
               file.find("/packrat/src/") != std::string::npos ||
+              file.find("/env/") != std::string::npos ||
```

Comment:
My main worry here is that someone might reasonably have things in an `/env/` folder that isn't a virtual environment (and so would be surprised to see RStudio failing to search that folder).